### DISCUSSION TITLE
Can we suggest multiline but give an option to only select the first line?

### DISCUSSION THREAD

**Discussion by etorth** [community/community#19753] [Copilot Conversations]
fantastic plug, using copilot.vim.
one question is, sometimes (actually most of the time), it suggests mutli-lines, but only first several line makes sense.
But looks I need to either accept all lines, or accept nothing.

Is there a way we can suggest multi-lines, then a shortcut like <c-n> only accept the first line?
Then it can suggest new multi-lines, and we decide to accept the firstline or not.

**Comment by edgarsi** [comment]
+1

Multi line suggestions are usually garbage. But their beginning is often on point. Often this happens when writing a new function - the proposed signature is correct but Copilot also suggests a multi line implementation with it.

**Comment by OldStarchy** [comment]
Some way of accepting a partial suggestion would be ideal. A number of times when writing a docblock copilot will complete the line but then include another `*/`. I thought it would be useful to have some kind of processing on the result text that would allow me to automatically strip `/\n\s*\*\/^/` but just having a way to accept a partial suggestion would be easier to use and (potentially) more flexible.

**Comment by BoD** [comment]
See also #12426
